### DISCUSSION TITLE
Curation report week of February 28

### DISCUSSION THREAD

**Discussion by khanspers** [wikipathways/GPML-Repo#50] [Curation reports]
Hi all,

Here's my curation report for this last week:

**Recent changes**: Lots of changes from Denise and others, nothing specific to report

**Under construction**: Mostly new pathways (many from students), some were test/tutorial pathways marked for deletion (deleted where applicable).

**Deleted**: Deleted a new test/tutorial pathways.

**Stubs**:

1. Does anyone know this author, or does anyone want to complete the pathway: [https://www.wikipathways.org/index.php/Pathway:WP5078](https://www.wikipathways.org/index.php/Pathway:WP5078)
I contacted the author back in Sep 2021 with no response. 

2. @marvinm2: Should this really be deleted?
[https://www.wikipathways.org/index.php/Pathway:WP5091](https://www.wikipathways.org/index.php/Pathway:WP5091)

**Up next: Lauren**

Kristina

**Comment by marvinm2** [comment]
Hey Kristina, 
The WP5091 should not be deleted. This is indeed not yet finished but will see further developments soon. Thanks for notifying me

**Reply by khanspers** [reply]
Thanks. I've removed the deletion tag.
